src/jit/compiler.h

```diff
@@ -598,6 +598,20 @@ class  LclVarDsc
         return (unsigned)(roundUp(lvExactSize, sizeof(void*)));
     }
 
+    bool                lvIsMultiregStruct()
+    {
+#if FEATURE_MULTIREG_ARGS_OR_RET
+#ifdef _TARGET_ARM64_
+        if ((TypeGet() == TYP_STRUCT) &&
+            (lvSize()  == 2 * TARGET_POINTER_SIZE))
+        {
+            return true;
+        }
+#endif  // _TARGET_ARM64_
+#endif  // FEATURE_MULTIREG_ARGS_OR_RET
```

Comment:
As we discussed, we may want to change/adapt this in future for x64/ux, but it is probably best to defer that to the work to unify the ABI handling.